ui/lib/kv/addon/components/page/list.hbs

```diff
@@ -46,16 +46,20 @@
       >
         <:content>
           <form {{on "submit" this.transitionToSecretDetail}} class="has-top-margin-m">
-            <InputSearch
-              @id="search-input-kv-secret"
-              @initialValue={{@pathToSecret}}
-              @onChange={{this.handleSecretPathInput}}
-              @placeholder="secret/"
-              data-test-view-secret
-            />
+            <div class="field">
+              <Hds::Form::TextInput::Base
```

Comment:
This input is missing a label, which is a problem from an accessibility standpoint. Some options we may have to fix this:
- convert the `Hds::Form::TextInput::Base` to `Hds::Form::TextInput::Field` and add a visible label (in which case, who should I speak to in order to know which text label to use? @Monkeychip do I need to speak with a Vault designer or you can provide one that I can use?)
- add some aria label or an invisible label (but I am not sure what would be the benefit, in this case, ti hide the label)

@MelSumner other suggestions or thoughts?